OpenRA.Mods.Common/Traits/Render/WithParachute.cs

```diff
@@ -0,0 +1,134 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.Graphics;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Renders a parachute on units.")]
+	public class WithParachuteInfo : UpgradableTraitInfo, ITraitInfo, IRenderActorPreviewSpritesInfo, Requires<RenderSpritesInfo>, Requires<IBodyOrientationInfo>
+	{
+		[Desc("The image that contains the parachute sequences.")]
+		public readonly string Image = "parach";
+
+		[Desc("Parachute opening sequence.")]
+		public readonly string OpeningSequence = "open";
+
+		[Desc("Parachute idle sequence.")]
+		public readonly string Sequence = "idle";
```

Comment:
You forgot all the `[SequenceReference("Image")]` during the refactor. See https://github.com/openra/openra/pull/7709.